src/pages/ExamsPage/data/reducer.js

```diff
@@ -35,15 +35,19 @@ const slice = createSlice({
     deleteExamAttempt: (state, attemptId) => ({
       ...state,
       attemptsList: state.attemptsList.filter(attempt => attempt.attempt_id !== attemptId.payload),
-    })
-    ,
+    }),
+    setCurrentExamIndex: (state, examId) => ({
+      ...state,
+      currentExamIndex: state.examsList.findIndex(exam => exam.id === examId.payload),
```

Comment:
This is setting the index based on the matching exam id. The reducer is the correct place for this logic to happen right?

Reply:
I think so, I'd probably remove the index from the name personally since the outer code doesn't need to know about an index just current exam.